src/webapi/date.rs

```diff
@@ -1,13 +1,117 @@
+    /// day is an integer value representing the day of the month (normally from 1 to 31)
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)
+    // https://www.ecma-international.org/ecma-262/6.0/#sec-date-year-month-date-hours-minutes-seconds-ms
+    pub fn new_with_date(year: i32, month: i32, day: i32) -> Self {
+        js!(
+            return new Date(@{year}, @{month}, @{day});
+        ).try_into().unwrap()
+    }
+    
+    /// Creates a JavaScript Date instance that represents a single moment in time.
+    /// Date objects are based on a time value that is the number of milliseconds since 1 January 1970 UTC.
+    /// 
+    /// year is an integer value representing the year. Values from 0 to 99 map to the years 1900 to 1999.
+    /// month is an integer value representing the month, beginning with 0 for January to 11 for December
+    /// day is an integer value representing the day of the month (normally from 1 to 31)
+    /// hours an integer value representing the minute segment of a time
+    /// seconds an integer value representing the second segment of a time
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)
+    // https://www.ecma-international.org/ecma-262/6.0/#sec-date-year-month-date-hours-minutes-seconds-ms
+    pub fn new_with_time(year: i32, month: i32, day: i32, hours: i32, minutes: i32, seconds: i32) -> Self {
+        js!(
+            return new Date(@{year}, @{month}, @{day}, @{hours}, @{minutes}, @{seconds});
+        ).try_into().unwrap()
+    }
+    
+    /// Creates a JavaScript Date instance that represents a single moment in time.
+    /// Date objects are based on a time value that is the number of milliseconds since 1 January 1970 UTC.
+    /// 
+    /// year is an integer value representing the year. Values from 0 to 99 map to the years 1900 to 1999.
+    /// month is an integer value representing the month, beginning with 0 for January to 11 for December
+    /// day is an integer value representing the day of the month (normally from 1 to 31)
+    /// hours an integer value representing the minute segment of a time
+    /// seconds an integer value representing the second segment of a time
+    /// milliseconds an integer value representing the millisecond segment of a time
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)
+    // https://www.ecma-international.org/ecma-262/6.0/#sec-date-year-month-date-hours-minutes-seconds-ms
+    pub fn new_with_time_and_millis(year: i32, month: i32, day: i32, hours: i32, minutes: i32, seconds: i32, milliseconds: i32) -> Self {
+        js!(
+            return new Date(@{year}, @{month}, @{day}, @{hours}, @{minutes}, @{seconds}, @{milliseconds});
+        ).try_into().unwrap()
+    }
```

Comment:
Hmm.. could we get rid of `new_with_time` and `new_with_date` for now and keep only this method? And perhaps rename it to something like `from_datetime`? (`datetime` name is also used in `chrono`; we can always add other methods later **if** there will really be a need for them)